pyccl/tests/test_background.py

```diff
@@ -24,7 +24,7 @@
 input_a_array = np.linspace(0.1, 1, 100)
 input_chi = ccl.background.comoving_radial_distance(COSMO, input_a_array)
 input_hoh0 = ccl.background.h_over_h0(COSMO, input_a_array)
-input_growth = ccl.background.growth_factor(COSMO, input_a_array)
+input_growth = ccl.background.growth_factor_unnorm(COSMO, input_a_array)
```

Comment:
Why did this switch to unnormalized?

Reply:
It should have been unnormalized all along. Otherwise, there is no way to make the `growth_factor_unnorm` function could work with normalized input arrays (since the normalization is lost).